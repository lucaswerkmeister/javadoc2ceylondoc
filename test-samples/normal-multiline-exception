/**
 * line 1
 *
 * @throws Exception if this
 *                   or that happens
 * @exception ArgumentException if something else happens
 */
