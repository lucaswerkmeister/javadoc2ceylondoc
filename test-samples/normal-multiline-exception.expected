"line 1"
throws(`class Exception`, "if this
                           or that happens")
throws(`class ArgumentException`, "if something else happens")
